src/components/FilterJobTitles.js

```diff
@@ -1,14 +1,23 @@
- import React from 'react'
+import React, { useState } from 'react'
+
+function FilterJobTitles({ jobTitle }) {
+  const [activeTitle, setActiveTitle] = useState()
+
+  const OnTitleChange = (title) => {
+    setActiveTitle(title);
+  }
```

Comment:
```suggestion
  const onTitleChange = (title) => {
    setActiveTitle(title);
  }
```

methods should be start with small letter